src/actions/actions.ts

```diff
@@ -511,6 +511,21 @@ export class ActionPayloadTypes {
         dataItems: PendingDataItem[]
     } = z
 
+    /**
+     * TODO
+     * 
+     * @param bcName
+     * @param dataItems
+     * @param assocValueKey
+     * @param selected
+     */
+    changeDataItemsFull: {
```

Comment:
This name is not valid; `changeDataItem` means changing data item in any way when this particular actions does not interact with data items at all and only handles additional, association widgets specific fields `_associate` and `_value`

Something like `setAllAssociationsFull` would work, but it appears there is already a similiar action implemented, check it out: [changeChildrenAssociations](https://github.com/tesler-platform/tesler-ui/blob/728d8b5ea2bc630e72bcc82462c9b79ae1db8f14/src/epics/data.ts#L535).
From looking at the code it handles exactly the same thing so this pull request could be collapsed to simply proprerly cover the existing action with JSDoc